From 20310cb1099b0c72cb446a3dab88f59d54378c3d Mon Sep 17 00:00:00 2001
From: Yuri Sevatz <yuri_sevatz@hotmail.com>
Date: Fri, 10 Apr 2015 00:10:35 -0400
Subject: [PATCH] Fix -lboost_date_time missing from CMakeLists.txt

---
 build/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/CMakeLists.txt b/build/CMakeLists.txt
index 5adb0f0..c91eb70 100644
--- a/build/CMakeLists.txt
+++ b/build/CMakeLists.txt
@@ -124,7 +124,7 @@ endif()
 # libraries
 find_package ( Threads REQUIRED )
 
-find_package ( Boost COMPONENTS system filesystem regex program_options REQUIRED )
+find_package ( Boost COMPONENTS system filesystem regex program_options date_time REQUIRED )
 if(NOT DEFINED Boost_INCLUDE_DIRS)
   message(SEND_ERROR "Boost is not found, or your boost version was bellow 1.46. Please download Boost!")
 endif()
